components/brave_wallet/renderer/v8_helper.h

```diff
@@ -40,9 +41,10 @@ v8::MaybeLocal<v8::Value> CallMethodOfObject(
     const base::StringPiece& method_name,
     std::vector<v8::Local<v8::Value>>&& args);
 
-v8::MaybeLocal<v8::Value> ExecuteScript(blink::WebLocalFrame* web_frame,
-                                        const std::string& script,
-                                        const std::string& name);
+v8::MaybeLocal<v8::Value> ExecuteScript(
+    blink::WebLocalFrame* web_frame,
+    const std::string& script,
+    const absl::optional<std::string>& name);
```

Comment:
let's also remove usage of kWalletStandardScript and then we can get rid of this name parameter.